models/config.yaml

```diff
@@ -223,7 +220,5 @@ llama-65b-gptq-3bit:
 .*(openorca-platypus2):
   instruction_template: 'OpenOrca-Platypus2'
   custom_stopping_strings: '"### Instruction:", "### Response:"'
-.*codellama:
-  rope_freq_base: 1000000
```

Comment:
It's good to leave the `truncation_length`s and `rope_freq_base`s as they are as they are used by transformers and ExLlama-v1 respectively.

Reply:
Okay, I thought that we automatically get `truncation_lengths` from n_ctx for all loaders.